docs/contracts.rst

```diff
@@ -109,18 +109,18 @@ Methods
 
 Each Contract Factory exposes the following methods.
 
-.. py:classmethod:: Contract.deploy(transaction=None, args=None)
```

Comment:
This documentation shouldn't be removed yet, but rather marked as deprecated in the documentation.

```
.. warning:: `Contract.deploy` has been deprecated in favor of .....
```

Reply:
:+1: to leaving and deprecating the old deploy method